R/as-data-frame.R

```diff
@@ -1,34 +1,64 @@
-CrunchDataFrame <- function (dataset) {
+CrunchDataFrame <- function (dataset, order) {
     ## S3 constructor method for CrunchDataFrame. terms.formula doesn't seem
     ## to like S4 subclass of environment
     stopifnot(is.dataset(dataset))
     out <- new.env()
     out$.crunchDataset <- dataset
+    out$.names <- c()
+    # set the order of the dataset based on order function, if no order is 
+    # given return all rows in the dataset in the order they appear
+    if (is.null(order)) {
```

Comment:
Absolutely, I should have dropped the if/else. Holdover from when there was something slightly different.